dask/dataframe/core.py

```diff
@@ -1591,23 +1635,26 @@ def unique(self):
         uniques : Series
         """
         return aca(self, chunk=methods.unique, aggregate=methods.unique,
-                   meta=self._meta, token='unique', series_name=self.name)
+                   meta=self._meta, token='unique', split_every=split_every,
+                   series_name=self.name)
 
     @derived_from(pd.Series)
-    def nunique(self):
-        return self.drop_duplicates().count()
+    def nunique(self, split_every=None):
+        return self.drop_duplicates(split_every=None).count()
 
     @derived_from(pd.Series)
-    def value_counts(self):
+    def value_counts(self, split_every=None):
         return aca(self, chunk=M.value_counts,
                    aggregate=methods.value_counts_aggregate,
-                   meta=self._meta.value_counts(), token='value-counts')
+                   combine=methods.value_counts_combine,
+                   meta=self._meta.value_counts(), token='value-counts',
+                   split_every=split_every)
 
     @derived_from(pd.Series)
-    def nlargest(self, n=5):
+    def nlargest(self, n=5, split_every=None):
```

Comment:
We will probably eventually want to put in some defaults for these.  For operations like nlargest I suspect it'll be in the tens and for operations like value_counts or some of the groupby computations I suspect it'll be around 10.